lxd-agent/metrics.go

```diff
@@ -0,0 +1,371 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/lxc/lxd/lxd/response"
+	"github.com/lxc/lxd/shared/api"
+	"github.com/lxc/lxd/shared/filesystem"
+	"github.com/pkg/errors"
+)
+
+var defMountPointsExcluded = regexp.MustCompile("^/(dev|proc|sys|var/lib/docker/.+)($|/)")
+var defFSTypesExcluded = regexp.MustCompile("^(autofs|binfmt_misc|bpf|cgroup2?|configfs|debugfs|devpts|devtmpfs|fusectl|hugetlbfs|iso9660|mqueue|nsfs|overlay|proc|procfs|pstore|rpc_pipefs|securityfs|selinuxfs|squashfs|sysfs|tracefs)$")
```

Comment:
@monstermunchkin for this one, from what i can see your exactly matching on a string, so you could use a slice of strings here and then use `shared.StringInSlice()`, to avoid needing to use regex here.